### ISSUE TITLE
arrivalAirportlocalTime is not defined

### ISSUE THREAD

**Issue by airmanopus** [airmanopus/drivinandflyin#143]
https://app.getsentry.com/steve-noonan-associates/drivin-and-flyin/issues/101261997/

```
ReferenceError: arrivalAirportlocalTime is not defined
  at XMLHttpRequest.flightTrackTransferComplete (/drivinandflyin/js/drivinandflyin.js:596:149)
```

**Comment by airmanopus** [comment]
Leaving this open for now. Probably need to take a closer look at the times we're displaying in the flight details.

**Comment by airmanopus** [comment]
Fixed on #154.
